doc/js_tutorials/js_assets/js_dnn_example_helper.js

```diff
@@ -0,0 +1,141 @@
+getBlodFromImage = function(inputSize, mean, std, swapRB, canvasId) {
+    let mat = cv.imread(canvasId);
+    let matC3 = new cv.Mat(mat.matSize[0], mat.matSize[1], cv.CV_8UC3);
+    cv.cvtColor(mat, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]),
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), swapRB);
+    mat.delete();
+    matC3.delete();
+    return input;
+}
+
+getBlodFromFrame = function(inputSize, mean, std, swapRB, frame) {
+    let matC3 = new cv.Mat(frame.matSize[0],frame.matSize[1],cv.CV_8UC3);
+    cv.cvtColor(frame, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]),
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), swapRB);
+    matC3.delete();
+    return input;
+}
+
+loadLables = async function(labelsUrl) {
+    let response = await fetch(labelsUrl);
+    let label = await response.text();
+    label = label.split('\n');
+    return label;
+}
+
+getNameFromUrl = function(url) {
+    const modelParts = url.modelUrl.split('/');
+    const modelPath = modelParts[modelParts.length-1];
+    const configParts = url.configUrl.split('/');
+    const configPath = configParts[configParts.length-1];
+    return {
+        modelPath: modelPath,
+        configPath: configPath
+    };
+}
+
+loadModel = async function(url) {
+    path = getNameFromUrl(url);
+    return new Promise((resolve) => {
+        // check if the model has been loaded before
+        if(modelLoaded.indexOf(path.modelPath) == -1){
+            document.getElementById('status').innerHTML = 'Loading model...';
+            utils.createFileFromUrl(path.modelPath, url.modelUrl, () => {
+                modelLoaded.push(path.modelPath);
+                // check if need to load config file
+                if(url.configUrl !== "") {
+                    utils.createFileFromUrl(path.configPath, url.configUrl, () => {
+                        resolve(path);
+                    });
+                } else {
+                    resolve(path);
+                }
+            });
+        } else {
+            resolve(path);
+        }
+    });
+}
+
+getModelType = function(modelPath) {
```

Comment:
Do we really need this logic?

Reply:
I just want to show the name and type of the running model after inference. I can delete this if not needed.